### ISSUE TITLE
run codeshow in vs2015 / windows10 - "MSAdvertisingJS, Version=8.1" missing

### ISSUE THREAD

**Issue by bschelling** [codefoster/codeshow#9]
I'm trying to run the codeshow app in Visual Studio 2015 on windows 10
It doesn't load - getting: Error MSB3774 Could not find SDK "MSAdvertisingJS, Version=8.1".

What can i do about it? I installed the Universal Ad Client SDK but to no avail...

**Comment by bschelling** [comment]
might there be a way to simply get rid of it? I'd just like to get the other solutions running...

**Comment by codefoster** [comment]
Yeah, try just pulling it out. The dependency and the code that refers to it.

jf

**Comment by codefoster** [comment]
I just haven't taken the time to upgrade this to Windows 10.

jf

**Comment by bschelling** [comment]
can you give me a hint of how to do this? the project doesn't load in vs2015 because of the error

**Comment by kbdavis07** [comment]
Go to references and remove msadvertisingJS then rebuild.

That worked for me using Windows 10, Visual Studio 2015
